specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -2319,6 +2322,25 @@
leIdentityCardNumber", "ChinaResidentIdentityCardPRCNumber", "HongKongIdentityCardHKIDNumber", "IndiaUniqueIdentificationAadhaarNumber", "IndiaPermanentAccountNumberPAN", "IndonesiaIdentityCardKTPNumber", "PhilippinesUnifiedMultiPurposeIDNumber", "BrazilCPFNumber", "BrazilNationalIDCardRG", "BrazilLegalEntityNumberCNPJ", "DenmarkPersonalIdentificationNumber", "BelgiumNationalNumber", "GermanyIdentityCardNumber", "IrelandPersonalPublicServicePPSNumber", "NetherlandsCitizenServiceBSNNumber", "NorwayIdentityNumber", "PortugalCitizenCardNumber", "CroatiaPersonalIdentificationOIBNumber", "CroatiaIdentityCardNumber", "CzechPersonalIdentityNumber", "GreeceNationalIDCard", "SouthAfricaIdentificationNumber", "InternationalClassificationofDiseasesICD10CM", "InternationalClassificationofDiseasesICD9CM", "AzurePublishSettingPassword", "AzureStorageAccountKeyGeneric", "SQLServerConnectionString", "AzureDocumentDBAuthKey", "AzureIAASDatabaseConnectionStringandAzureSQLConnectionString", "AzureIoTConnectionString", "AzureRedisCacheConnectionString", "AzureSAS", "AzureServiceBusConnectionString", "AzureStorageAccountKey", "EUPhoneNumber", "EUMobilePhoneNumber", "EUPassportNumber", "EUNationalIdentificationNumber", "EUSocialSecurityNumberSSNorEquivalentID", "EUDriverLicenseNumber", "EUTaxIdentificationNumberTIN", "EUGPSCoordinates", "AustralianCompanyNumber", "AustriaVATNumber", "UkrainePassportNumberInternational", "UkrainePassportNumberDomestic", "FinlandEuropeanHealthInsuranceNumber", "FranceHealthInsuranceNumber", "RussianPassportNumberDomestic", "RussianPassportNumberInternational", "NewZealandBankAccountNumber", "NewZealandInlandRevenueNumber", "AustralianBusinessNumber", "PolishREGONNumber", "SwissSocialSecurityNumberAHV", "GermanyValueAddedTaxNumber", "JapaneseMyNumberPersonal", "JapaneseMyNumberCorporate", "NetherlandsValueAddedTaxNumber", "ItalyValueAddedTaxNumber", "HungarianValueAddedTaxNumber", "FranceValueAddedTaxNumber", "BelgiumValueAddedTaxNumber", "NewZealandDriverLicenseNumber", "NewZealandSocialWelfareNumber", "AustriaTaxIdentificationNumber", "LuxemburgNationalIdentificationNumberNaturalpersons", "NetherlandsTaxIdentificationNumber", "SloveniaTaxIdentificationNumber", "LuxemburgNationalIdentificationNumberNonnaturalpersons", "PolandTaxIdentificationNumber", "BulgariaUniformCivilNumber", "AustriaIdentityCard", "CyprusTaxIdentificationNumber", "CyprusIdentityCard", "EstoniaPersonalIdentificationCode", "GreekTaxidentificationNumber", "HungaryPersonalIdentificationNumber", "HungaryTaxidentificationNumber", "ItalyFiscalCode", "LithuaniaPersonalCode", "MaltaIdentityCardNumber", "SpainTaxIdentificationNumber", "UKUniqueTaxpayerReferenceNumber", "FranceTaxIdentificationNumber", "GermanyTaxIdentificationNumber", "PortugalTaxIdentificationNumber", "RomaniaPersonalNumericalCode", "SlovakiaPersonalNumber", "CroatiaNationalIDCardNumber", "SloveniaUniqueMasterCitizenNumber", "SwedenTaxIdentificationNumber", "LatviaPersonalCode", "SpainDN", "All", "Default"]
```

Comment:
I think we should have a review over the names of the types.
There's more alignment we should drive like: 2-letter country code prefix (`USSocialSecurityNumber` vs. `CanadaSocialInsuranceNumber`), use of acrynoms (`USSocialSecurityNumber` vs. `USIndividualTaxpayerIdentificationNumberITIN`), etc.

Reply:
Types have been updated